### PR TITLE
Add Vue and React examples for the NumberBox

### PR DESCRIPTION
**Pull request by DokaRus**

### CODE REVIEW

File: concepts/05 Widgets/NumberBox/10 Handle the Value Change Event.md [hunk 1]
```diff
@@ -33,6 +33,44 @@ By default, the value of the **NumberBox** is changed when the <a href="https://
         // ...
     })
 
+##### Vue
+
+    <!-- tab: App.vue -->
+    <template> 
+        <DxNumberBox
+            :value="20"
+            valueChangeEvent="keyup"
```
Comment (RomanTsukanov) [inline concepts/05 Widgets/NumberBox/10 Handle the Value Change Event.md]:
```suggestion
            value-change-event="keyup"
```

File: concepts/05 Widgets/NumberBox/10 Handle the Value Change Event.md [hunk 2]
```diff
@@ -62,9 +100,7 @@ To process a new **NumberBox** value, you need to handle the value change event.
     import { DxNumberBoxModule } from "devextreme-angular";
     // ...
     export class AppComponent {
-        numberBox_valueChanged (e) {
-            let previousValue = e.previousValue;
-            let newValue = e.value;
+        numberBox_valueChanged ({ previousValue, value: newValue }) {
```
Comment (RomanTsukanov) [inline concepts/05 Widgets/NumberBox/10 Handle the Value Change Event.md]: It looks like in Angular and Vue we should show two-way binding and in React state changing.

File: concepts/05 Widgets/NumberBox/10 Handle the Value Change Event.md [hunk 3]
```diff
@@ -76,6 +112,48 @@ To process a new **NumberBox** value, you need to handle the value change event.
         // ...
     })
 
+##### Vue
+
+    <!-- tab: App.vue -->
+    <template> 
+        <DxNumberBox @valueChanged="numberBoxValueChanged" />
```
Comment (RomanTsukanov) [inline concepts/05 Widgets/NumberBox/10 Handle the Value Change Event.md]:
```suggestion
        <DxNumberBox @value-changed="numberBoxValueChanged" />
```

File: concepts/05 Widgets/NumberBox/10 Handle the Value Change Event.md [hunk 4]
```diff
@@ -76,6 +112,48 @@ To process a new **NumberBox** value, you need to handle the value change event.
...
+
+    <!-- tab: App.js -->
+    import React from 'react';
+
+    import NumberBox from 'devextreme-react/number-box';
+
+    class App extends React.Component {
+        render() {
+            return (
+                <NumberBox onValueChanged="numberBoxValueChanged" />
```
Comment (RomanTsukanov) [inline concepts/05 Widgets/NumberBox/10 Handle the Value Change Event.md]:
```suggestion
                <NumberBox onValueChanged={this.numberBoxValueChanged} />
```

File: concepts/05 Widgets/NumberBox/10 Handle the Value Change Event.md [hunk 5]
```diff
@@ -76,6 +112,48 @@ To process a new **NumberBox** value, you need to handle the value change event.
...
```
Comment (AlexJameson) [inline concepts/05 Widgets/NumberBox/10 Handle the Value Change Event.md]:
```suggestion
                <NumberBox onValueChanged={this.numberBoxValueChanged} />
```

File: concepts/05 Widgets/NumberBox/20 Keyboard Support.md [hunk 6]
```diff
@@ -65,6 +65,59 @@ You can implement a custom handler for a key using the [registerKeyHandler(key,
...
+
+    <!-- tab: App.js -->
+    import React from 'react';
+
+    import NumberBox from 'devextreme-react/number-box';
+
+    class App extends React.Component {
+        render() {
+            return (
+                <NumberBox onInitialized="registerKeyHandlers" />
```
Comment (AlexJameson) [inline concepts/05 Widgets/NumberBox/20 Keyboard Support.md]: If function call fix is applied above, apply it here as well.

**Review by AlexJameson — APPROVED** [review]

File: concepts/05 Widgets/NumberBox/20 Keyboard Support.md [hunk 7]
```diff
@@ -65,6 +65,59 @@ You can implement a custom handler for a key using the [registerKeyHandler(key,
         // ...
     })
 
+##### Vue
+
+    <!-- tab: App.vue -->
+    <template> 
+        <DxNumberBox @initialized="registerKeyHandlers"
```
Comment (RomanTsukanov) [inline concepts/05 Widgets/NumberBox/20 Keyboard Support.md]: We usually call `registerKeyHandler` on `mounted`/`componentDidMount()`:
https://js.devexpress.com/Documentation/20_1/Guide/Widgets/Button/Keyboard_Support/

**Review by RomanTsukanov — CHANGES_REQUESTED** [review]

File: concepts/05 Widgets/NumberBox/10 Handle the Value Change Event.md (hunk 2 above)
Comment (DokaRus) [reply concepts/05 Widgets/NumberBox/10 Handle the Value Change Event.md]: [please notice](https://github.com/DevExpress/devextreme-documentation/pull/331#discussion_r380236985)

Comment (RomanTsukanov) [reply concepts/05 Widgets/NumberBox/10 Handle the Value Change Event.md]: Let's do here what you've done on the Calendar PR.

Comment (DokaRus) [reply concepts/05 Widgets/NumberBox/10 Handle the Value Change Event.md]: Done

**Review by RomanTsukanov — APPROVED** [review]

**Pull request merged**